Makefile

```diff
@@ -39,55 +39,55 @@ functional: ginkgo
 ifeq (${RUNTIME},)
 	$(error RUNTIME is not set)
 else
-	./ginkgo -v functional/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
+	chronic ./ginkgo -v functional/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
 endif
 
 docker: ginkgo
 ifeq ($(RUNTIME),)
 	$(error RUNTIME is not set)
 else
-	./ginkgo -v -focus "${FOCUS}" -skip "${SKIP}" ./integration/docker/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
+	chronic ./ginkgo -v -focus "${FOCUS}" -skip "${SKIP}" ./integration/docker/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
```

Comment:
oh, yeah, you are probably right :-( mumble.... any thoughts on if there is anything else we can do or other options (`chronic` doesn't have a regular poll type output ...)

Reply:
We used to have ginkgo without verbose, which suppress a lot of the messages we currently see in the log. But those are helpful when we need to debug why a test fail...

Taking a look at the [ginkgo docs](http://onsi.github.io/ginkgo/), I found these options which might help:
```
--succinct

Succinct silences much of Ginkgo’s more verbose output. Test suites that succeed basically get printed out on just one line! Succinct is turned off, by default, when running tests for one package. It is turned on by default when Ginkgo runs multiple test packages.

--trace

If present, Ginkgo will print out full stack traces for each failure, not just the line number at which the failure occurs.
```


Btw, the cri-o tests do not send much output, I think we can remove the chronic from those.

Reply:
There is always the magic nul byte hack - if Travis/Jenkins are simply looking for any type of characters to avoid timing out, we could start a background process that writes a "`\0`" periodically. Something like one of:

- 
  ```
  $ utfout -b 1s '' -r -1
  ```

-
  ```
  $ while true; do printf "\0";sleep 1; done
  ```